services/app-api/utils/constants/constants.ts

```diff
@@ -16,6 +16,8 @@ export const error = {
   NO_TEMPLATE_NAME: "Must request template for download.",
   INVALID_TEMPLATE_NAME: "Requested template does not exist or does not match.",
   NOT_IN_DATABASE: "Record not found in database.",
+  UNABLE_TO_COPY:
+    "Unable to copy over report if todays date is the same period and year as previous report.",
```

Comment:
maybe we want to pass this by product? My only update would be
"Unable to copy over report if todays date is in the same period and year as a previous report."

Reply:
This is a message that only gets sent back as an api error to help us debug, so its really only help to us for debugging purposes. That said, I can make the change!